client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -21,46 +25,53 @@ type Props = ProductStoreBaseProps & {
 	siteId: number | null;
 };
 
-const Includes = () => (
-	<ul>
-		<li>Real-time backups as you edit</li>
-		<li>10GB of cloud storage</li>
-		<li>30-day activity log archive</li>
-		<li>Unlimited one-click restores from the last 30 days</li>
-		<li>WooCommerce order and table backups</li>
-		<li>Redundant cloud backups on our global network</li>
-		<li>Priority support</li>
-	</ul>
-);
+const DescriptionList: React.FC< { items?: TranslateResult[] } > = ( { items } ) => {
+	if ( ! items || ! items.length ) return null;
+
+	return (
+		<ul>
+			{ items.map( ( item ) => (
+				<li key={ item.toString() }>{ item }</li>
```

Comment:
Why `item.toString()` instead of `item`?

Reply:
So `item` here is actually a `TranslationResult` which is `ReactChild` not compatible with `key`

Reply:
Oh yes, it's `key` prop.